src/player.ts

```diff
@@ -437,6 +437,20 @@ export class Player extends NativeInstance<PlayerConfig> {
     return PlayerModule.getAvailableVideoQualities(this.nativeId);
   };
 
+  /**
+   * Sets the video quality.
+   * ONLY works on Android. No effect on iOS and tvOS devices.
+   */
+  setVideoQuality = (qualityId: String) => {
+    if (Platform.OS !== 'android') {
+      console.warn(
+        `[Player ${this.nativeId}] Method setVideoQuality is not available for iOS and tvOS devices. Only Android devices.`
+      );
+    } else {
+      PlayerModule.setVideoQuality(this.nativeId, qualityId);
+    }
```

Comment:
nit: I think it would be easier to read if we add an early return inside the `if` statement and extract the content of the `else` statement, like here:

```suggestion
    if (Platform.OS !== 'android') {
      console.warn(
        `[Player ${this.nativeId}] Method setVideoQuality is not available for iOS and tvOS devices. Only Android devices.`
      );
      return;
    }
    PlayerModule.setVideoQuality(this.nativeId, qualityId);
```